Corona-Warn-App/src/main/java/de/rki/coronawarnapp/ui/view/TestResultSectionView.kt

```diff
@@ -26,30 +27,39 @@ constructor(
     attrs: AttributeSet? = null,
     defStyleAttr: Int = 0
 ) : ConstraintLayout(context, attrs, defStyleAttr) {
+    private lateinit var testResultSectionHeadline: TextView
+    private lateinit var testResultSectionContent: TextView
+    private lateinit var testResultSectionRegisteredAt: TextView
+    private lateinit var testResultSectionStatusIcon: ImageView
 
     init {
         inflate(context, R.layout.view_test_result_section, this)
         context.withStyledAttributes(attrs, R.styleable.TestResultSection) {
-            test_result_section_headline.text =
+            testResultSectionHeadline = findViewById(R.id.test_result_section_headline)
+            testResultSectionContent = findViewById(R.id.test_result_section_content)
+            testResultSectionRegisteredAt = findViewById(R.id.test_result_section_registered_at_text)
+            testResultSectionStatusIcon = findViewById(R.id.test_result_section_status_icon)
```

Comment:
`TestResultSectionViewBinding.bind(this)`might be helpful here to avoid this boilerplate code

Reply:
I was not sure if view binding works the same way for custom views. Thanks for this! :) I will update it in the afternoon.

Reply:
Adjusted it